Source/CombatExtended/CombatExtended/Verbs/Verb_LaunchProjectileCE.cs

```diff
@@ -348,9 +359,128 @@ public virtual void ShiftTarget(ShiftVecReport report, bool calculateMechanicalO
                     }
                     else if (currentTarget.Thing is Pawn)
```

Comment:
Since you sometimes need to access this as a `Pawn`, may as well do the cast here.
`else if (currentTarget.Thing is Pawn victim)`
This is more robust against future changes as `victim` is only defined within the `if` block, and the if block only runs if `victim` is not `null`.